packages/trader/src/AppV2/Components/AccumulatorStats/accumulator-stats.scss

```diff
@@ -0,0 +1,191 @@
+@mixin accumulatorSlideToNext {
+    animation: slideToNext 0.24s ease-in-out forwards;
+    @keyframes slideToNext {
+        from {
+            transform: translateX(-30px);
+        }
+        to {
+            transform: translateX(0px);
+        }
+    }
+}
+
+@mixin accumulatorUnderlineAnimation($color: var(--core-color-solid-green-600)) {
+    animation: underlineAnimation 640ms ease-in-out forwards;
+    background-color: $color;
+    @keyframes underlineAnimation {
+        0% {
+            transform: scaleX(0);
+            opacity: 0;
+        }
+        50% {
+            transform: scaleX(1);
+            opacity: 1;
+        }
+        50.1% {
+            transform: scaleX(1);
+            opacity: 1;
+        }
+        100% {
+            transform: scaleX(0);
+            opacity: 0;
+        }
+    }
+}
+
+.accumulators-stats {
+    background: var(--general-main-1);
+    padding: var(--core-spacing-400);
+    &__container {
+        display: flex;
+        justify-content: space-between;
+        &__heading {
+            margin: var(--core-spacing-300) var(--core-spacing-400);
+            background: transparent;
+            border: none;
+            border-bottom: var(--core-spacing-75) dashed;
+            padding: var(--core-spacing-50);
+        }
+        &__divider {
+            border-right: var(--core-spacing-75) solid var(--core-color-opacity-black-100);
+            margin: var(--core-spacing-50) var(--core-spacing-400);
+        }
+        .slide-right {
+            @include accumulatorSlideToNext;
+        }
+
+        &__stats {
+            margin: var(--core-spacing-300) var(--core-spacing-50);
+            display: flex;
+            width: calc(100% - 48px - 9px - 40px);
+            justify-content: space-between;
+            gap: var(--core-spacing-400);
+            position: relative;
+            overflow-x: auto;
+            &::-webkit-scrollbar {
```

Comment:
Will it work in all browsers (and on IOS)? 
Usually we use the combination of `scrollbar-width:none` and `&::-webkit-scrollbar {display: none;}`
I don't remember exactly, but each css option cover +-half of browsers.